www/includes/easyparliament/templates/html/static/help.php

```diff
@@ -149,24 +122,39 @@
                 by scraping various pages on the official parliament.uk site. The official site
                 stopped maintaining a list of PPSs in January 2009, which means our scraper could
                 no longer find and work out the information. Without a central list of
-                Parliamentary Private Secretaries, there isn&rsquo;t a lot we can do.</p>
+                Parliamentary Private Secretaries, sadly there isn&rsquo;t a lot we can do.</p>
 
                 <h3 class="js-toc-item" id="searching">
-                    How can I search more smartly?
+                    Can I search TheyWorkForYou?
                 </h3>
 
-                <p>Click &lsquo;Advanced search&rsquo; under the box on the
-                search results page, and you&rsquo;ll see a number of
-                options for refining your results. But you can do
-                everything that Advanced Search does, and even refine your
-                search further, right from the search box.</p>
+				<p>
+					You certainly can  — you'll see the Search box at the top of every page. If you just want to 
+					search within a specific MP's speeches, there's a search box on each MP's page, too. 
+				</p>
+				
+				<p>
+					But that might not be detailed enough for your needs, so there's more. 
+				</p>
+				
+				<p>
+					Click &lsquo;Advanced search&rsquo; under the box on the search results page, 
+					and you&rsquo;ll see a number of options for refining your results. 
+
+				</p>
+				
+				<p>
+				That makes it easy to find what you want. Want to level up? If you&rsquo;re using TheyWorkForYou on a regular basis, 
+				it&rsquo;s worth knowing that you can do everything Advanced Search does,
+				and even refine your search further, right from the search box.
+				</p>
 
                 <ul>
                     <li>To search for an <strong>exact phrase</strong>, use quotes (""). For
                     example, to find only documents which contain the exact phrase <a
                     href="/search/?q=%22Hutton+Report%22">"Hutton Report"</a>. Quotes can also
-                    be used to prevent stemming, so searching for &ldquo;horse&rdquo; will omit results for
-                    horses or horsing.
+                    be used to prevent stemming, so searching for &ldquo;refugee&rdquo; will omit results for
+                    refugees or reefuge.
```

Comment:
don't think reefuge is a word? If it was meant to be refuge, this isn't a good example, I'm afraid, as refugee stems to Zrefuge, sure, but refuge stems to Zrefug, so you don't get overlapping results (and reefuge stems to Zreefug).